### PR TITLE
Status effect stufff

### PR DESCRIPTION
**Pull request by liach**

### CODE REVIEW

**Review by haykam821 — APPROVED** [review]

File: mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping [hunk 1]
```diff
@@ -1,4 +1,8 @@
 CLASS net/minecraft/class_1293 net/minecraft/entity/effect/StatusEffectInstance
+	FIELD field_21830 hiddenEffect Lnet/minecraft/class_1293;
+		COMMENT The effect overwritten when merging effects. Duration decreases with
+		COMMENT this one, and always have a longer duration and lower amplifier compared
+		COMMENT to this effect.
```
Comment (Pyrofab) [inline mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping]: This doesn't seem clear enough. Could you give examples of how this is used ?

Comment (liach) [reply mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping]: this is rather internal for status effects. not really user oriented

Comment (liach) [reply mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping]: just upgrade (merge) one long effect with low amplifier with one short effect with high amplifier

Comment (Pyrofab) [reply mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping]: so when the short effect with a strong amplifier runs out, the long effect with a weak amplifier takes over ? I think you should just say that.

Comment (liach) [reply mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping]: https://minecraft.gamepedia.com/Java_Edition_1.15.2_Pre-Release_1#Gameplay

Comment (Pyrofab) [reply mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping]: I meant saying that in the comment, as I think it's not clear enough right now.

Comment (liach) [reply mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping]: Oh shoot, @Pyrofab please do not block prs for trivial things like this... This is not public, just some impl detail and never used by mods. You are discouraging people from documenting private fields and I am disagreeing over this (you've also asked for too much on profiler mappings)

Comment (Pyrofab) [reply mappings/net/minecraft/entity/effect/StatusEffectInstance.mapping]: Because as everyone knows, my suggestions are legally binding.

**Review by Pyrofab — APPROVED** [review]

**Review by YanisBft — APPROVED** [review]

**Review by Juuxel — APPROVED** [review]

**Pull request merged**